code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -128,6 +128,9 @@
 			seen_revived_enemy = TRUE
 			raise_alert("[name] reports intruder [target] has returned from death!")
 			depotarea.list_remove(target, depotarea.dead_list)
+		if(!atoms_share_level(src, target) && prob(20))

```

Comment:
This check stops people cheesing depot mobs by hiding in lockers. Previously, if players A and B walked into a room with the mob, and the mob aggroed on A, A could hide in a locker, resulting in the mob standing next to the locker while B safely attacked them. With this change, attempting this strategy is risky, as the depot mob may switch targets at any moment.